packages/widgets/src/dns-hole/controls/component.tsx

```diff
@@ -149,7 +169,17 @@ const ControlsCard = (
                 {t(`widget.dnsHoleControls.controls.${isEnabled ? "enabled" : "disabled"}`)}
               </Badge>
             </UnstyledButton>
-            <ActionIcon disabled={!isEnabled} size={20} radius="xl" top="2.67px" variant="default" onClick={open}>
+            <ActionIcon
+              disabled={!isEnabled}
+              size={20}
+              radius="xl"
+              top="2.67px"
+              variant="default"
+              onClick={() => {
+                setSelectedIntegration([integrationId]);
```

Comment:
Use plural

Reply:
I unmarked the "resolve".
Either make the function accept only single element instead of an array, or make it plural, just because you use it with a single item doesn't make it singular. you still pass an array to it and It could be used as is otherwise.

Like I mentioned before though, if you made the state for single items to begin with and assume all integration when that state is unset, then you can set all of them to single and won't need to do specific things as such.